proc_IR.py

```diff
@@ -7,10 +7,11 @@
 save_npz = False
 #}}}
 coherence_pathway = {'ph1':0,'ph2':1}
+# this should not be part of the PR
```

Comment:
we are editing the correlation alignment function which is used in our exp data processing and so to test that it works on ACTUAL data I add the processing for IR and enhancement